src/gemm_v2.cpp

```diff
@@ -484,7 +487,12 @@ miopenStatus_t CallGemm(const Handle& handle,
                 rocblas_datatype::rocblas_datatype_f32_r,
                 rocblas_gemm_algo::rocblas_gemm_algo_standard,
                 0,
-                0);
+#if USE_GEMM_FLAGS_FP16_ALT_IMPL
+                gemm_desc.gfx90aaltimpl ? rocblas_gemm_flags_fp16_alt_impl : 0
```

Comment:
Proposed approach: GEMM Invokers read `gfx90aFp16alt` attribute (bool) from the instance of `AnyInvokeParams` passed to it. This bool should be then s forward it down to this function. And here let's use it instead of `gemm_desc.gfx90aaltimpl`.

AFAIU, unlike [this code](https://github.com/ROCmSoftwarePlatform/MIOpen/pull/1226/files#r744003897), we do not need to check either for gfx90a target (and maybe FP16 datatype check is not required as well).

Reply:
Done!